api/src/test/java/uk/gov/dhsc/htbhf/claimant/testsupport/PaymentCycleVoucherEntitlementTestDataFactory.java

```diff
@@ -7,12 +7,26 @@
 import java.util.List;
 
 import static java.util.Collections.singletonList;
-import static uk.gov.dhsc.htbhf.claimant.testsupport.VoucherEntitlementTestDataFactory.aValidVoucherEntitlement;
-import static uk.gov.dhsc.htbhf.claimant.testsupport.VoucherEntitlementTestDataFactory.aVoucherEntitlementWithEntitlementDate;
-import static uk.gov.dhsc.htbhf.claimant.testsupport.VoucherEntitlementTestDataFactory.aVoucherEntitlementWithZeroVouchers;
+import static uk.gov.dhsc.htbhf.claimant.testsupport.VoucherEntitlementTestDataFactory.*;
 
 public class PaymentCycleVoucherEntitlementTestDataFactory {
 
+    public static PaymentCycleVoucherEntitlement aPaymentCycleVoucherEntitlementWithPregnancyVouchers() {
+        return buildDefaultPaymentCycleVoucherEntitlement()
+                .voucherEntitlements(List.of(
```

Comment:
Can we use `Collections.singletonList()` here, it's much clearer that it should only have a single element, `List.of` is great for anything else though.

Reply:
Done.

Reply:
Looks like you've done it below but not to this one???